docs/02-guides/translate-your-application.mdx

```diff
@@ -0,0 +1,303 @@
+```
+
+Fortunately, this is correctly handled by `react-intl` if you use
+[`defineMessages`](https://formatjs.io/docs/react-intl/api/#definemessagesdefinemessage)
+combined with the
+[`useIntl`](https://formatjs.io/docs/react-intl/api/#useintl-hook) hook or the
+[`injectIntl`](https://formatjs.io/docs/react-intl/upgrade-guide-3x/#new-useintl-hook-as-an-alternative-of-injectintl-hoc)
+HOC.
+
+```mdx-code-block
+<Tabs>
+<TabItem value="useIntl">
+```
+
+You can learn more about hooks in the
+[react documentation](https://reactjs.org/docs/hooks-intro).
+
+```jsx
+import { defineMessages, useIntl } from "react-intl";
+
+const messages = defineMessages({
+  ariaLabel: {
+    id: "screen-reader-icon-title",
+    defaultMessage: "Icon title displayed for screen readers",
+  },
+});
+
+const MyComponentWithHook = (props) => {
+  const intl = useIntl();
+  return (
+    <span class="icon" aria-label={intl.formatMessage(messages.ariaLabel)} />
+  );
+};
+```
+
+```mdx-code-block
+</TabItem>
+<TabItem value="injectIntl">
+```
+
+You can learn more about <abbr title="Higher-Order Components">HOC</abbr>'s in
+the [react documentation](https://reactjs.org/docs/higher-order-components).
+
+```jsx
+import { defineMessages, injectIntl } from "react-intl";
+
+const messages = defineMessages({
+  ariaLabel: {
+    id: "screen-reader-icon-title",
+    defaultMessage: "Icon title displayed for screen readers",
+  },
+});
+
+const MyComponentWithHOC = injectIntl(({ intl, ...props }) => {
+  return (
+    <span class="icon" aria-label={intl.formatMessage(messages.ariaLabel)} />
+  );
+});
+```
+
+```mdx-code-block
+</TabItem>
+</Tabs>
+```
+
+## Translate what's in your components
+
+Now that you have defined what should be translated in your application, you
+actually need to translate your application. This is a two-step process:
+
+1. Run the following script that will fetch all your translatable strings in
+   your application and gather them in a JSON file located in
+   `translations/[locale].json`
+
+   ```shell
+   $ front-commerce translate <path> --locale <locale>
+   ```
+
+   :::tip ProTip™
+
+   The script has already been defined in the
+   [skeleton template](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/main/skeleton/package.json?ref_type=heads#L13),
+   and can be used directly with `pnpm run translate`. You can change the
+   default locale to accommodate your store.
+
+   :::
+
+2. Translate the strings available in those files, and you are good to go 🙂
+
+   :::tip ProTip™
+
+   You can use tools like [BabelEdit](https://www.codeandweb.com/babeledit) to
+   help translate and keep your translations in sync.
+
+   :::
+
+### Translations fallback
+
+With `react-intl` translations are usually grouped into a single file. In our
+case, we would expect them to be in `translations/[locale].json`. But in
+Front-Commerce's case, we don't want you to be troubled by translations that are
```

Comment:
**nit:** Duplicate meaning between `In our case` / `In Front-Commerce's case`.  I think this could be reworded as 
```suggestion
With `react-intl` translations are usually grouped into a single file. In our
case, we would expect them to be in `translations/[locale].json`, but we
don't want you to be troubled by translations that are
```